### PR TITLE
Scrape and Parse SputnikMusic.com for trending albums 

### PR DESCRIPTION
**Pull request by lukecoy**
Parse SputnikMusic.com webpage for trending albums

a lot of boilerplate

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "trendy",
+  "version": "1.0.0",
+  "description": "",
```
Comment (lukecoy) [inline package.json]: hmm

File: package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "main": "server.js",
+  "scripts": {
+    "start": "nodemon lib/server.js --exec babel-node",
+    "build": "babel lib -d dist",
+    "serve": "node dist/server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (lukecoy) [inline package.json]: yeaa about that

**Comment by lukecoy** [comment]
should probably take care of the readme

File: lib/sputnik-scraper.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import jsdom from 'jsdom';
+
+export default function scrapeSputnikWebpage() {
+  return new Promise((resolve, reject) => {
+    jsdom.env({
+      url: 'http://sputnikmusic.com',
+      scripts: ["http://code.jquery.com/jquery.js"],
```
Comment (lukecoy) [inline lib/sputnik-scraper.js]: Make this a local reference

**Comment by lukecoy** [comment]
aahhh style guide   

move all this random existential stuff to github issues

File: package.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "trendy",
+  "version": "1.1.0",
+  "description": "Find cool stuff, fast",
+  "main": "lib/server.js",
+  "scripts": {
```
Comment (lukecoy) [inline package.json]: todo - refactor these scripts a bit

File: lib/sputnik/parseSputnikWebpage.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+// Will throw if unable to parse HTML
+export default function parseSputnikWebpage(window) {
+  const $ = window.$;
```
Comment (lukecoy) [inline lib/sputnik/parseSputnikWebpage.js]: if we could not use jquery that'd be nice but not super necessary

**Comment by ethanjurman** [comment]
:trophy: best code, 2016.

**Comment by nicolasmccurdy** [comment]
:cool:

**Comment by lukecoy** [comment]
you guys should star my repo and work with me on it

**Pull request merged**